Code/Framework/AzFramework/Tests/CameraInputTests.cpp

```diff
@@ -106,7 +106,8 @@ namespace UnitTest
 
         //! This is approximately Pi/2 * 1000 - this can be used to rotate the camera 90 degrees (pitch or yaw based

```

Comment:
Super nitpick - should this just be a regular comment? Seems a bit weird to have doxygen markup in a cpp and in a test. I guess it's not doing any harm since I don't imagine doxygen will ever be run on this file, just seemed a bit odd.

Reply:
Ah yes good point, oversight on my part, will fix